### PR TITLE
Backend/change question ai methods

### PR DESCRIPTION
**Pull request by Integraluminium**

### CODE REVIEW

File: backend/src/requests/user-content-request/user-content-request.service.ts [hunk 1]
```diff
@@ -293,9 +294,15 @@ export class UserContentRequestService {
...
+    if (data.useAI && data.useAI != AI_OPTIONS.NONE && userExist) {
       const isPro = await this.userService.isProUser(userId);
-      this.requestAI(data.content, question.groupId, userId, isPro).then();
+      this.requestAI(
+        data.content,
+        question.groupId,
+        userId,
+        isPro,
+        data.useAI,
+      ).then();
```
Comment (Jozys) [inline backend/src/requests/user-content-request/user-content-request.service.ts:305]: Is this .then needed?

**Review by Jozys — APPROVED** [review]
Looks fine.

**Review by Jozys — DISMISSED** [review]
Looks fine.

**Review by benniloidl — APPROVED** [review]

**Review by Jozys — APPROVED** [review]
Looks fine.

**Pull request merged**